### PR TITLE
Add additional alert for data using a time window which reflects what is on the government dashboard

### PR DESCRIPTION
**Pull request by amyhughes**
## What does this change?
Pamela is keen to make a version of this email alert available to people in the newsroom outside of her team, but for this new version she would like the numbers to match what is on the [government dashboard](https://coronavirus.data.gov.uk/). 

The government dashboard states:
"Data shown are cases by specimen date and because these are incomplete for the most recent dates, the period represented is the 7 days ending 5 days before the date when the website was last updated."

This time window lags the time window for the email we have been sending until now, which represents the 7 days ending the last day for which the API returns data. We still think there is value in sending alerts based on the most up-to-date but possibly incomplete data, but since the caveat is difficult to understand we do not want to send these alerts to people who aren't in the know.

As such, here we modify the stack to have two lambdas, one which triggers an email with the "verified" data which matches the dashboard, and one which triggers an email with the most up-to-date but "unverified" data.

### Testing the code
For the verified email type, working out exactly how many days to subtract from the latest date for which we see data at the time of running was a little fiddly, but we have settled on 4 days since:

- our lambda runs at 8am
- the government data is updated at 4pm
- running locally before 4pm (e.g. at 10am today),  we see:
image
- this matches the numbers on the dashboard:
image

For the unverified email type, running locally setting the environment variable `EMAIL_TYPE=UNVERIFIED` and artificially lowering the `PERCENTAGE_CHANGE_THRESHOLD` to `50.0`, we see results up to two days ago:

image

### Testing the infrastructure
I have fully deployed the lates build for this branch and run the lambdas to receive emails

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -57,11 +65,11 @@ def save_metric_definitions(metric_names):
 
 
 def get_ltla_populations():
-    url = "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/populationandmigration/populationestimates" \
-          "/datasets/populationestimatesforukenglandandwalesscotlandandnorthernireland" \
-          "/mid2019april2019localauthoritydistrictcodes/ukmidyearestimates20192019ladcodes.xls"
+    url = "https://www.ons.gov.uk/file?uri=%2fpeoplepopulationandcommunity%2fpopulationandmigration%2fpopulationestimates" \
```
Comment (amyhughes) [inline src/main.py:68]: This is moving to use more up to date population figures for Lower Tier Local Authorities, published [here](https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/populationestimatesforukenglandandwalesscotlandandnorthernireland). Motivated by the per 100,000 metric being slightly off what was shown on the [government dashboard](https://coronavirus.data.gov.uk/details/cases)

File: src/main.py [hunk 2]
```diff
@@ -109,28 +117,39 @@ def get_metric_per_100000_nhs_region(metric, nhs_populations_df, nhs_region_name
...
+        "six_days_before": upper_bound - pd.Timedelta(days=6),
+        "seven_days_before": upper_bound - pd.Timedelta(days=7),
+        "thirteen_days_before": upper_bound - pd.Timedelta(days=13)
     }
 
 def date_dependent_column_names(metric_name, dates):
     return {
-        "week_before": f'{metric_name}-{dates["thirteen_days_before"].strftime("%m-%d-%Y")}-to-{dates["seven_days_before"].strftime("%m-%d-%Y")}',
-        "last_week": f'{metric_name}-{dates["six_days_before"].strftime("%m-%d-%Y")}-to-{dates["latest"].strftime("%m-%d-%Y")}'
+        "week_before": f'{metric_name}-{dates["thirteen_days_before"].strftime("%d-%m-%Y")}-to-{dates["seven_days_before"].strftime("%d-%m-%Y")}',
```
Comment (amyhughes) [inline src/main.py:130]: Moving away from American style dates

File: package.json [hunk 3]
```diff
@@ -2,10 +2,10 @@
   "name": "uk-coronavirus-data-alerts",
   "version": "0.0.1",
   "dependencies": {
-    "@guardian/cdk": "^15.1.0",
+    "@guardian/cdk": "guardian/cdk#make-it-possible-to-define-multiple-lambdas-per-stack",
```
Comment (amyhughes) [inline package.json:5]: TODO: publish the cdk library with [our fix](https://github.com/guardian/cdk/pull/781)

File: src/main.py (hunk 1 above)
Comment (philmcmahon) [reply src/main.py:68]: nice fix

**Review by philmcmahon — APPROVED** [review]
nice work

File: package.json (hunk 3 above)
Comment (jacobwinch) [reply package.json:5]: Feel free to merge https://github.com/guardian/cdk/pull/781 whenever you're ready - it should be released automatically

**Pull request merged**